### ISSUE TITLE
No App Launcher

### ISSUE THREAD

**Issue by TungstenCarbideProjectile** [oscartbeaumont/ElectronPlayer#7]
The app works for me quite well but i can only launch it from the snap store. It did not add an icon to my start menu or desktop.

**Comment by oscartbeaumont** [comment]
Sorry, It has taken me so long to reply, I didn't see this issue until now. It seems like you are getting issue #11 which has been fixed in the newest release, please update your snap and try again. If it is not #11 please give me your environments details. Eg. what distro, distro version, system arch, snap version, electronplayer version?

**Comment by oscartbeaumont** [comment]
I am closing this issue due to inactivity. If your problem has not been solved please leave a comment and I can reopen the issue.

**Comment by TungstenCarbideProjectile** [comment]
Oscar im sorry i did not reply sooner as well i have been bogged down at work and home. I love the program it is great.  

Updating snap store did not seem to alleviate the issue on ubuntu 18.04 or 19.04 on a x86 architecture Intel processor not sure of the snap version or electron version at the moment. 

I created a desktop launcher and to my great excitement electron auto populated with all the info needed for a perfect desktop launcher. I then created a .desktop entry in applications folder and it is now available through my start menu.  

Basically those two things just didn't happen automatically and if it had been fixed through issue #11 that's great.  Just wanted to update you with info and thank you for getting back to me. 

You can leave this topic closed or request more information from me i will get back to you quicker next time. If you want me go hey you versions and try to replicate i can upon your request. 

Again i love the program and thank you for the help.
